datastore/db/management/commands/create_data_package.py

```diff
@@ -86,3 +112,86 @@ def flatten_grant(in_grant):
             data_all_fp.write(json.dumps(data_all, indent=options["indent"]))
 
         spinner.stop()
+
+
+CREATE_RELATED_ORGIDS = """
+DROP TABLE IF EXISTS related_orgids;
+
+CREATE TABLE related_orgids AS (    
+    with org_name AS (SELECT                                                   
+        data -> 'linked_orgs' ->> 0 AS cannonical_orgid, 
+        string_agg(data ->> 'name', '||') AS name,
+        string_agg(org_id, '||') AS name_org_id
+    FROM additional_data_orginfocache GROUP BY 1
+    )
+    SELECT                                                   
+        linked_org ->> 0 org_id,
+        data -> 'linked_orgs' related_orgids,
+        data -> 'linked_orgs' ->> 0 AS cannonical_orgid,
+        max(name) AS name,
+        max(name_org_id) AS name_org_id
+    FROM 
+        additional_data_orginfocache orgi 
```

Comment:
should this be orgid?

Reply:
nope that is an shorter alias of `additional_data_orginfocache`